extensions/compose/src/extension.ts

```diff
@@ -65,6 +65,11 @@ export async function activate(extensionContext: extensionApi.ExtensionContext):
   const composeGitHubReleases = new ComposeGitHubReleases(octokit);
   const composeDownload = new ComposeDownload(extensionContext, composeGitHubReleases, os);
 
+  // add a command to display the onboarding page
+  const openOnboardingCommand = extensionApi.commands.registerCommand('compose.openComposeOnboarding', async () => {
```

Comment:
Is `compose.openComposeOnboarding` command declaration is missing in package.json?

Reply:
it's not missing, it's internal (from compose to compose)